docs/core/testing/unit-testing-mstest-runner-intro.md

```diff
@@ -55,3 +55,61 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
 
 </Project>
 ```
+
+## Configurations and filters
+
+### .runsettings
+
+The MSTest runner supports the [runsettings](unit-testing-platform-runsettings.md) through command line option `--settings`, for instance:
+
+using `dotnet run`
+
+```bash
+dotnet run --project Contoso.MyTests -- --settings config.runsettings
+```
+
+using `dotnet exec`
+
+```bash
+dotnet exec Contoso.MyTests.dll --settings config.runsettings
+```
+
+or
+
+```bash
+dotnet Contoso.MyTests.dll --settings config.runsettings
+```
+
+using the executable
+
+```bash
+Contoso.MyTests.exe --settings config.runsettings
+```
+
+### Tests filter
+
+Seamesly you can provite the tests [filter](./selective-unit-tests#mstest-examples) using the command line option `--filter`, for instance:
```

Comment:
This link is incorrect, please fix this - not sure where it should point to?

Reply:
This one https://learn.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=mstest#mstest-examples looks in the same folder or?